content/pairing.md

```diff
@@ -0,0 +1,248 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.6.0
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Pairing Jupyter notebooks and MyST-NB
+
+## What you'll do
+This guide will keep a Jupyter notebook synced _or paired_ between
+`.ipynb` and `.md`. 
+
+## What you'll learn
+- The difference between Jupyter's json format and MyST-NB's markdown
+  format
+- The benefits and drawbacks of json and markdown
+- How to keep `.ipynb` and `.md` files in sync
+
+## What you'll need
+- [Jupyter](https://jupyter.org/)
+- [Jupytext](https://jupytext.readthedocs.io/en/latest/index.html)
+
+---
+## Background
+
+The [NumPy tutorials](https://github.com/numpy/numpy-tutorials) are
+reviewed and executed as [MyST-NB](https://myst-nb.readthedocs.io/)
+notebooks.  Content is easier to review in this markdown format. You can
+keep your `.ipynb` in sync with the content on NumPy tutorials. The
+NumPy tutorials use
+[Jupytext](https://jupytext.readthedocs.io/en/latest/index.html) to
+convert your `.ipynb` file to [MyST
+Markdown](https://github.com/mwouts/jupytext/blob/master/docs/formats.md#myst-markdown)
+format.
+
+Jupyter notebooks are stored on your disk in a
+[json](https://www.json.org/json-en.html) format. The json format is
```

Comment:
I would suggest adding this link 

`https://nbformat.readthedocs.io/en/latest/format_description.html`

instead of the json one to this paragraph, might be more useful for less experienced users.

```suggestion
Jupyter notebooks are stored on your disk in a
[json format](https://nbformat.readthedocs.io/en/latest/format_description.html). The json format is
```